pwem/convert/image_handler.py

```diff
@@ -261,11 +261,20 @@ def getDimensions(self, locationObj):
             location = self._convertToLocation(locationObj)
             fn = location[1]
             ext = pwutils.getExt(fn).lower()
-            
+
+            # Local import to avoid import loop between ImageHandler and Ccp4Header.
+            from pwem.convert import Ccp4Header
```

Comment:
I think this import should be moved to the top...and also converted to a local import (since it is internal in the submodule):
```python
from .headers import Ccp4Header
```